.ruby-version

```diff
@@ -0,0 +1 @@
+2.0.0-p353
```

Comment:
yes. I would love to omit this. I had some weird problems with my vagrant environment and had to install a newer Ruby version to be able to work on this fork. I got my environment set up so I could fork it and contribute, but I didn't realize it would effect any of my merge requests.

What's the best practice to omit this? Do I simply delete it out of the .ruby-version file and do another merge request?

Thanks!